contents.json

```diff
@@ -16516,6 +16516,23 @@
       "screenshots": [
         "https://raw.githubusercontent.com/popshootjapan/WeiWallet-iOS/master/resources/cover_img.png"
       ]
+    },
+    {
+      "title": "Bitwarden",
+      "category-ids": [
+        "password"
+      ],
+      "description": "An easy way for individuals, teams, and business organizations to store, share, and sync sensitive data.",
+      "source": "https://github.com/bitwarden/mobile",
+      "homepage": "https://bitwarden.com",
+      "itunes": "https://itunes.apple.com/app/bitwarden-free-password-manager/id1137397744",
+      "license": "gpl-3.0",
+      "stars": 426,
+      "screenshots": [
+        "https://camo.githubusercontent.com/b2dff900f45c86c85e1c65a8576356a87971fa9e/68747470733a2f2f692e696d6775722e636f6d2f52374832746b512e706e67","https://camo.githubusercontent.com/3d2118c643c885c7c9c12fee554ece577af36c7b/68747470733a2f2f692e696d6775722e636f6d2f33424f315763672e706e67"
```

Comment:
Could you keep only the screenshot for iOS and replace `camo.*` URL to direct URLs. You can use this link https://i.imgur.com/3BO1Wcg.png.